qemu/tests/migration_multi_host_cancel.py

```diff
@@ -48,39 +53,35 @@ def check_guest(self):
             for vm in self.vms:
                 try:
                     vm = env.get_vm(vm)
+                    stress_kill_cmd = params.get("stress_kill_cmd",
+                                                 "killall -9 stress")
                     session = vm.wait_for_login(timeout=self.login_timeout)
-                    session.sendline("killall -9 cpuflags-test")
+                    session.sendline(stress_kill_cmd)
+                    vm.reboot()
```

Comment:
pls double check it, in my understanding, stress should last 300sec, no need to kill it , and reboot will cancel it as well.